frontend/src/pages/AccountSend.vue

```diff
@@ -15,10 +15,40 @@
     <!-- Send form -->
     <q-form v-else @submit="onFormSubmit" class="form" ref="sendFormRef">
       <!-- Identifier -->
-      <div v-if="!advancedMode">Recipient's ENS or CNS name</div>
+      <div v-if="!advancedMode">Recipient's ENS name or address</div>
       <div v-else>Recipient's ENS name, CNS name, transaction hash, address, or uncompressed public key</div>
       <base-input v-model="recipientId" :disable="isSending" placeholder="vitalik.eth" lazy-rules :rules="isValidId" />
 
+      <!-- Identifier, advanced mode tooltip -->
+      <q-checkbox
```

Comment:
It doesn't seem to be possible to use this feature unless the receiving address has set up Umbra, which goes counter to the purpose of this power user feature. I think we'd want to disable the check when the user has this checked.

<img width="913" alt="Screen Shot 2021-10-05 at 1 54 56 PM" src="https://user-images.githubusercontent.com/1479802/136077661-252f4335-be74-43f9-bd17-15e1aa7f9c22.png">

<img width="928" alt="Screen Shot 2021-10-05 at 1 54 09 PM" src="https://user-images.githubusercontent.com/1479802/136077664-cc8ae6a0-26ab-4b64-93c9-f80dd981af2d.png">

Reply:
Actually, this feature seems not to work at all. I sent 201 DAI to benrink.eth with the checkbox checked, and they went to the stealth registry key instead.

Reply:
Whoops, big bug! Should be fixed in e9828e6d918af440d680c05edda38f5ff6636ecc, let me know if it's still not working as expected

Reply:
Whoops, big bug! Should be fixed in e9828e6d918af440d680c05edda38f5ff6636ecc, let me know if it's still not working as expected